accessibility-oscilloscope/package/signal-processing/signal-processing.mk

```diff
@@ -0,0 +1,19 @@
+################################################################################
+#
+# signal-processing
+#
+################################################################################
+
+SIGNAL_PROCESSING_VERSION = v0.0.1
+SIGNAL_PROCESSING_SOURCE = $(SIGNAL_PROCESSING_VERSION).tar.gz
+SIGNAL_PROCESSING_SITE = https://github.com/accessibility-oscilloscope/signal-processing/archive/refs/tags
+
+define SIGNAL_PROCESSING_BUILD_CMDS
+	$(MAKE) $(TARGET_CONFIGURE_OPTS) -C $(@D) signal-processing
```

Comment:
same question here, is there actually a Makefile doing something? I thought it was all python and shell scripts